### PR TITLE
Remove inactive maintainers

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

**Review by bitsondatadev — APPROVED** [review]

**Comment by mosabua** [comment]
Lets wait for confirmation with merge @bitsondatadev

**Comment by mosabua** [comment]
All three confirmed in slack/email/linkedin that they are no longer working as maintainers. This should be merged together with permissions changes that @martint will work on with me. We will merge the PR at that stage.

**Pull request merged**
